dataland-frontend/src/utils/UpdateObjectUtils.ts

```diff
@@ -0,0 +1,45 @@
+export type objectType = { [key: string]: string | object };
```

Comment:
A type name should start with capital letter.

Reply:
ok